util/auroMigrate.md

```diff
@@ -79,16 +79,22 @@ This function takes up to three arguments in this order:
 1. Path to the newly created directory `{ String }`
 1. `--no-demo` flag `{ Boolean }`
 
-For example. Migrating from a legacy repo, `flight`, assuming that both repos are in the same root directory, and the new repo is named `auro-flight`, the command would be the following:
+For obvious reasons, you cannot have two directories named the same. The migration command accepts any directory path, e.g. if the newly creates repo was in a sub-directory from where the legacy repo is, the command could be the following:
 
 ```
-$ auroMigrate flight auro-flight
+$ auroMigrate auro-flight migration/auro-flight
```

Comment:
Since simply renaming the original version would still work I think we should highlight that example still rather than removing it. Personally, I find it quicker than creating a new folder and moving the original.